maintainers/maintainer-list.nix

```diff
@@ -12010,6 +12010,12 @@
     githubId = 248016;
     name = "Solène Rapenne";
   };
+  sree = {
+    email = "sreec@canva.com";
+    github = "pathway27";
+    githubId = 1563316;
+    name = "Sreekara Chelamalla";
+  };
```

Comment:
Please separate this change into a separate commit titled `maintainers: add sree`.